TSResourceFileType.js

```diff
@@ -81,15 +81,16 @@ TSResourceFileType.prototype.name = function() {
  * @return {TSResourceFile} a resource file instance for the
  * given path
  */
-TSResourceFileType.prototype.newFile = function(pathName) {
+TSResourceFileType.prototype.newFile = function(pathName, options) {
     var file = new TSResourceFile({
         project: this.project,
         pathName: pathName,
+        locale: options.locale || this.project.sourceLocale,
```

Comment:
should be:
```
(options && options.locale) || this.project.sourceLocale
```
In case this plugin is being run by an older version of loctool that doesn't support the second parameter yet.